homeassistant/components/frontend/__init__.py

```diff
@@ -32,9 +32,10 @@
 
 CONF_THEMES = 'themes'
 CONF_EXTRA_HTML_URL = 'extra_html_url'
+CONF_EXTRA_HTML_URL_ES5 = 'extra_html_url_es5'
 CONF_FRONTEND_REPO = 'development_repo'
 CONF_JS_VERSION = 'javascript_version'
-JS_DEFAULT_OPTION = 'es5'
+JS_DEFAULT_OPTION = 'auto'
```

Comment:
I wonder if we should keep this on `es5` for one more release and have people be able to try out `auto`.